### ISSUE TITLE
Anyway to change the spacing between QMChatCell avatarView and ContainerView?

### ISSUE THREAD

**Issue by astanton** [QuickBlox/quickblox-ios-sdk#521]
I've been digging into the customization of the chat, and I'm really liking how much control I have over the look and feel of the chat view.  However there is one thing I haven't been able to figure out from looking at the docs and code at this point - is there anyway to change the spacing between the avatar image and the message view?  

I see that the QMImageView and QMChatContainerView are what make up the avatarView and containverView, however I would like to add some more space between the two if possible.  

And on the flip side, for the outgoing messages, I am not displaying an avatar over there and the message also seems to be pretty close to the edge of the view.  I was wondering if there was a general control that would add more spacing around the chat message view that would help out with both of these instances.

Thanks!

**Comment by Pro100Andrey** [comment]
Hi @astanton, please check: 
<img width="1040" alt="ocell" src="https://cloud.githubusercontent.com/assets/1109916/16257371/d1e5d678-3860-11e6-8cbf-20fb5b9ea101.png">

**Comment by VitaliyGuru** [comment]
@astanton  Any updates?

**Comment by astanton** [comment]
Yeah sorry for not replying, I actually did figure this out in an earlier post that I made over on the QMChatViewController git repo and actually made another post with a suggestion on possibly making those constraints IBOutlets that we can configure so we don't have to go in and modify the xib files.  

https://github.com/QuickBlox/QMChatViewController-ios/issues/68

I have gone in and modified them, however if I ever upgrade then I'm going to have to go in and change it again, which is where the configuration with an outlet could come in handy.

Thanks for the help!
